pkg/generated/clientset/versioned/fake/clientset_generated.go

```diff
@@ -41,9 +41,10 @@ func NewSimpleClientset(objects ...runtime.Object) *Clientset {
 		}
 	}
 
-	fakePtr := testing.Fake{}
-	fakePtr.AddReactor("*", "*", testing.ObjectReaction(o))
-	fakePtr.AddWatchReactor("*", func(action testing.Action) (handled bool, ret watch.Interface, err error) {
+	cs := &Clientset{}
```

Comment:
I'm not sure if this is a right change. :thinking: 

Running `hack/update-all.sh` generated these changes.

Reply:
Try running `make update` instead - that uses a docker build container with all the right k8s repo versions.

Reply:
Fixed!